src/web/handlers/TapGestureHandler.ts

```diff
@@ -87,7 +87,10 @@ export default class TapGestureHandler extends GestureHandler {
   private startTap(): void {
     this.clearTimeouts();
 
-    this.waitTimeout = setTimeout(() => this.fail(), this.maxDurationMs);
+    this.waitTimeout = setTimeout(
+      () => this.fail(),
+      this.maxDurationMs
+    ) as unknown as number;
```

Comment:
Same

Reply:
Done in [f6a46e5](https://github.com/software-mansion/react-native-gesture-handler/pull/2873/commits/f6a46e59bf7ebe489416d68fc2700e18d5bb3d05)